src/decoder_wildmidi.cpp

```diff
@@ -131,6 +131,10 @@ WildMidiDecoder::WildMidiDecoder(const std::string file_name) {
 		config_file = "wildmidi.cfg";
 		found = FileFinder::Exists(config_file);
 	}
+#elif SWITCH
+	// Only wildmidi paths, no timidity because it was never used on Switch
+	config_file = "./wildmidi.cfg";
+	found = FileFinder::Exists(config_file);
 #elif __ANDROID__
```

Comment:
Could you add after your check
```cpp
	if (!found) {
		config_file = "/switch/easyrpg-player/wildmidi.cfg";
		found = FileFinder::Exists(config_file);
}
```
?

Just as a fallback in case somebody decides to use a different folder or directly ship a game with it.